hadoop-ozone/dist/src/main/compose/upgrade/test.sh

```diff
@@ -44,4 +41,4 @@ run_test ha     non-rolling-upgrade 1.4.1 "$OZONE_CURRENT_VERSION"
 
 generate_report "upgrade" "$ALL_RESULT_DIR"
 
-exit "$RESULT"
+exit "$?"
```

Comment:
Please keep `exit "$RESULT"`.  The variable is set in `run_test`.

Reply:
oh thanks for pointing out, "$?" will only keep the latest command result...

let me change it back and also try to align commit message style